keyboards/wilba_tech/wt_rgb_backlight.c

```diff
@@ -1551,20 +1551,14 @@ void backlight_effect_indicators(void)
     }
 
 #if defined(RGB_BACKLIGHT_NK65)
-    if ( g_indicator_state & (1<<USB_LED_CAPS_LOCK) )
-    {
+    if ( g_indicator_state & (1<<USB_LED_CAPS_LOCK) ) {
         IS31FL3733_set_color( 7+64-1, 0, 255, 0 );
     } else {
         IS31FL3733_set_color( 7+64-1, 0, 0, 0 );
     }
-    if ( g_indicator_state & (1<<USB_LED_SCROLL_LOCK) )
-    {
+    if ( IS_LAYER_ON(1) ) {
```

Comment:
Does this affect PCBs with Scroll and Numlock indicators?

Reply:
Ahh, I see, this only affects the NK65?

Reply:
Yep it basically makes the scroll lock led and the numlock led layer indicators!